components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsUserPreferences/src/IsAnalyticsUserPreferences.jsx

```diff
@@ -320,8 +320,8 @@ class IsAnalyticsUserPreferences extends Widget {
                                     </td>
                                     <td style={{ padding: 15 }}>
                                         <Button
-                                            color="primary"
-                                            variant="flat"
+                                            color="#ef6c00"
```

Comment:
Not a good practice you define your own colors like this. Try to use colors from the theme as much as possible. Please add a TODO for this.